### DISCUSSION TITLE
✅ Don't load drop-cap css unless drop caps are actually being used on a page.

### DISCUSSION THREAD

**Discussion by joehdevwp** [elementor/elementor#22339] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

From a recent Elementor support ticket I received this response: 

"When you have the Improved CSS Loading experiment active, Elementor will add widget-specific CSS to every first instance of the widget on pages only once, instead of including basic styles for all widgets on a single big CSS file elementor-frontent.css. Meaning it will only load the CSS that is required for that page, and that widget, only once. If you don't use that widget elsewhere, it won't load the basic CSS for that widget. This means that your pages will load lighter and perform better depending on the page you are on. The basic CSS for the Text Editor includes styles for the widget's Drop-cap feature."

Note the phrase, "it will only load the CSS that is required for that page" This is false. The CSS Loading experiment loads the following css on each and every page with a text editor widget regardless of whether or not drop caps are being used:

```html
<style>
/*! elementor - v3.12.2 - 23-04-2023 */
.elementor-widget-text-editor.elementor-drop-cap-view-stacked .elementor-drop-cap {
    background-color:#69727d;
    color:#fff
}

.elementor-widget-text-editor.elementor-drop-cap-view-framed .elementor-drop-cap {
    color:#69727d;
    border:3px solid;
    background-color:transparent
}

.elementor-widget-text-editor:not(.elementor-drop-cap-view-default) .elementor-drop-cap {
    margin-top:8px
}

.elementor-widget-text-editor:not(.elementor-drop-cap-view-default) .elementor-drop-cap-letter {
    width:1em;
    height:1em
}

.elementor-widget-text-editor .elementor-drop-cap {
    float:left;
    text-align:center;
    line-height:1;
    font-size:50px
}

.elementor-widget-text-editor .elementor-drop-cap-letter {
    display:inline-block
}
</style>
```

I understand the intention is to optimize the page speed, but to load css for a feature is that so rarely used seems completely contradictory. 

### Describe the solution you'd like

Only load the inline drop-cap css when dropcaps are actually being used on the page. Elementor is already bloated enough without loading css that isn't being used at all. 

### Additional context

I realize this is a small issue, and perhaps makes very little impact on page speed, but little impacts add up, and Elementor is already known for using excessive dom elements and bloated code compared to its competitors. If this CSS were for a feature that was commonly used perhaps it would be acceptable.

**Comment by nicholaszein** [comment]
Hello there @joehdevwp !

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Core v3.28.0! 🥳 

Now Drop Caps and other control-specific styles only load when in use!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
